token-swap/program/src/processor.rs

```diff
@@ -3971,7 +4065,8 @@ mod tests {
         let host_fee_account = spl_token::state::Account::unpack(&pool_account.data).unwrap();
         let owner_fee_account =
             spl_token::state::Account::unpack(&accounts.pool_fee_account.data).unwrap();
-        let total_fee = host_fee_account.amount * host_fee_denominator / host_fee_numerator;
+        let total_fee = owner_fee_account.amount * host_fee_denominator
+            / (host_fee_denominator - host_fee_numerator);
```

Comment:
This made me realize that host fees probably should never be 100%

Reply:
haha thankfully it's just a shorthand for the test